static-site/app/styles/globals.css

```diff
@@ -139,3 +139,14 @@ p.wideParagraph {
   line-height: var(--tightLineHight);
   margin: 20px 0 0;
 }
+
+
+/* For blog posts */
+
+.anchor {
+   opacity: 0;
+}
+
+.anchor:hover {
+  opacity: 100;
+}
```

Comment:
> this was the easiest thing to do in absence of a blog-specific static CSS file.

Can this be added to the [app/blog/[id]/styles.modules.css](https://github.com/nextstrain/nextstrain.org/blob/master/static-site/app/blog/%5Bid%5D/styles.module.css) as a blog-specific styling?

Reply:
No, CSS modules are not "static"; the classes get dynamically modified to something like

```css
.styles_anchor__1sHC6 {
   opacity: 0;
}

.styles_anchor__1sHC6:hover {
  opacity: 100;
}
```

and `className={styles.hover}` can't be used since it's not a React/Next.js component.

Reply:
Ah gotcha...I'm guessing it'd be possible to pass in `styles.hover` as a param to `parseMarkdown` but maybe that's just overcomplicating things.

Reply:
I've reorganized the commits and wrote more about this in c562b67f

Reply:
> I'm guessing it'd be possible to pass in styles.hover as a param to parseMarkdown but maybe that's just overcomplicating things.

Not asking for a change, I was curious what this would look like: https://github.com/nextstrain/nextstrain.org/commit/a20a9aed7d5c7947663ee6608bd170c72653fbae.

Reply:
Oh nice! I'll open a separate PR to consider that.

Reply:
#1092